packages/formation-react/src/components/AccessibleLink/AccessibleLink.unit.spec.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import AccessibleLink from './AccessibleLink';
+import { createIcon } from '../../helpers/link-utils';
+
+describe('Widget <AccessibleLink />', () => {
+  describe('utility functions', () => {
```

Comment:
Can you move this to another file? One for all the utility tests would be great.